content/en/blog/2022/otel-unplugged-kubecon-na-22

```diff
@@ -0,0 +1,25 @@
+---
+title: {{ replaceRE "[-_]" " " .Name | OTel Unplugged at KubeCon NA 2022! }}
+linkTitle: OTel Unplugged at KubeCon NA 2022! 
+date: 2022-10-05
+author: >- # If you have only one author, then add the single name on this line in quotes.
+  [Austin Parker](https://github.com/lightstep) (Lightstep),
+draft: false # TODO: remove this line once your post is ready to be published
+# canonical_url: http://somewhere.else/ # TODO: if this blog post has been posted somewhere else already, uncomment & provide the conancial URL here.
+---
```

Comment:
I think that you'll need to clean up the front matter -- I'm surprised this even builds. E.g., for the title, just include the title text.

Reply:
I can't see the blog post in the preview. Maybe it's because of the `{{ ... }}` directives in the title.